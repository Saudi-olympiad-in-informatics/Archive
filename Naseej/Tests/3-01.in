148165 501
56762 143494 171436447
128497 93233 538646359
7856 136905 401210742
1892 135439 763684752
35335 83323 114906601
81063 115174 276598586
104406 147718 583379208
9010 69406 624457095
36426 46794 887310180
9552 85768 898238816
31378 79644 146351973
87691 72743 433801258
36939 115958 386020584
10323 3300 406623768
105929 33815 58415203
26861 80651 518374044
37593 145297 452871086
75686 16486 731134035
83788 77205 698017095
41391 110787 364307689
65372 28747 262445976
143128 113637 879876601
116262 92540 586407313
91000 135116 864699371
121651 5466 102312428
25927 103895 359876054
117420 70156 635556280
36471 38197 232096863
90257 131185 374720624
27783 34937 976700621
42757 48488 87150139
29665 99747 491150096
111927 65313 380895953
110583 136999 329280517
126317 7223 232788304
30575 35267 287081411
24542 124374 7046807
95308 112765 647517241
116489 139289 320541837
43915 81337 952131993
27179 140957 858532594
92647 12346 659630233
32579 145807 868952769
128639 131839 965539928
141188 73518 346002544
124566 96135 855216776
147367 62031 373511099
110011 51100 645674333
52599 67081 384141476
30502 108841 91143436
49165 132788 598010346
59714 4221 931970227
53435 58863 784213670
14800 65600 49806725
61659 131523 365645668
147549 95630 225962138
110998 1127 98964467
77473 50042 171623316
138998 88581 431359192
96618 127804 905839477
43774 10931 61613289
68759 121132 532916418
62239 17524 364682425
119331 84130 901242032
104121 84393 120956507
23877 50160 409319814
122740 89435 95056129
41910 36646 549190598
32424 40418 215533536
122593 121211 267192249
39777 24833 914945171
87032 125654 778377997
77340 7590 84362477
112668 2678 665718948
44811 17350 7181705
139710 28073 438702776
24636 63868 910027734
40712 14469 625164395
31807 75913 79982302
53219 32246 78476253
17271 111486 341283130
91179 54037 601701187
112822 13488 803412710
84120 19947 103660834
138930 40864 5826223
61948 98415 667227365
118849 145777 519396930
91804 100985 151554376
49910 40736 985483793
21500 122563 991970599
90578 124818 926162755
64821 114348 373535451
10618 100568 372769517
140544 140198 123385453
103808 79307 819754713
93477 93822 217116977
21123 1826 209003220
17780 42686 295558397
32941 145155 335514747
21633 110660 95768837
128917 84500 227786396
6399 68081 159182256
117525 120029 538696654
33397 22620 758819012
147086 12801 398080439
119893 15407 8295494
41041 76927 216648799
112986 87734 588761638
132674 81231 544817416
21180 52001 910128690
51021 75285 776318455
59605 74665 633088292
131565 94741 370422822
124346 146662 568747695
54866 66898 385837603
95110 53869 934696432
127662 87055 402374068
140616 91531 921984486
135732 80405 6574244
92013 65748 906749545
9716 81079 178667608
66523 48129 907465068
77500 113775 437709219
53299 64696 487367657
4413 89515 889231782
65396 139402 118178707
37987 35170 727744886
145759 87131 91360297
10119 19829 657512977
51509 146507 297071828
13392 118854 484191555
55821 67887 227023070
42101 117392 881210708
73641 60248 114164826
138796 112262 931793682
18074 6025 620002132
35758 35600 696701014
8655 53654 835704122
28572 135944 526683218
37523 42551 701056197
77091 115359 299728576
63504 60762 270101899
113655 86749 874363685
76104 57465 915053265
55240 131614 565837785
61399 58956 330214336
7689 118908 380707761
133568 18935 831415524
40765 62414 957183622
128550 53831 636182628
98735 38950 472881338
91727 34743 218798640
16376 116533 475836195
86707 56484 548877192
59088 146182 879167301
114070 13959 343480741
66570 71887 632762224
4449 76528 473311979
118366 56983 790625012
63970 94806 533672510
60593 116444 466475282
123366 129571 886708724
73081 110691 413062350
834 105474 537155171
85812 39279 605891475
25770 32091 851809962
80977 47952 364375219
92175 67557 350653254
22360 62603 772940607
92531 59832 116220736
94521 96411 741449970
16813 136943 424660241
38650 127967 937153194
19235 114415 106021468
88254 78798 479937307
2599 43454 112689754
103946 73730 134522692
42168 75167 245838415
6831 23410 11942772
62475 113563 363059572
82744 13008 571842506
45481 130968 19099351
66445 67 802533025
86442 97765 249047151
19256 19043 749307834
144138 1956 412325374
110193 24458 479986228
19652 94619 764400465
113876 127512 738513901
30799 17406 297370452
146357 68302 401446958
116213 45441 330773563
134823 8156 447148475
133850 50204 168418574
53053 66162 564460401
89547 60368 499034881
80023 32648 262793960
54755 89682 854454247
69890 72912 735511409
5869 62940 580150768
87368 90699 301111950
91688 54109 423377350
83178 69750 115299918
57659 39432 598254736
139593 47338 945546604
54660 133506 106930068
6042 19248 624242694
89959 111353 23236981
20070 138620 550220863
97617 62321 115177793
110031 135001 138719224
114535 2177 150565870
1073 16683 557218905
38095 57667 810024141
98604 38420 557387670
12445 76829 24663927
147496 25686 42773529
99043 119820 404794777
16199 25664 746416331
18802 129133 413924926
13075 45925 125171419
123879 6534 939303410
57537 77737 553678432
110773 106645 87362915
30271 25191 241820641
78281 73321 391198820
28869 102798 267368394
104541 128331 105666986
73264 28449 307752068
110161 55924 970729634
120889 122811 578956019
13818 13085 312818655
99400 48690 449482097
133984 139871 25804921
70903 82430 87885744
142709 31481 292474145
26766 80400 884327502
46084 6003 608152103
52857 96979 317615835
58732 65187 942578355
130459 84908 664315361
31612 109705 687459709
52622 86808 722366431
23424 38452 583706201
104760 112642 484327551
42208 27305 217771233
90335 38099 327946104
56560 110599 504472124
58126 80954 368695924
35439 92330 504144546
21647 43904 287736582
96457 69975 595107524
85472 114408 902584916
37383 50344 25575066
26327 110463 740180163
109584 138179 836690495
75677 47746 873220305
79051 71329 10084595
63200 127269 310836510
59672 78055 260045939
78153 61371 162898745
47900 117075 596704155
138535 114785 102290851
39423 44388 989447476
11338 118459 754892508
6640 107791 680097374
48535 44928 996085544
121826 44468 455555790
92942 37106 60610755
106113 97070 178325878
86802 128723 24320962
40980 130173 174077026
5204 44042 578643825
6239 98340 145411821
21546 49614 949035324
127767 99743 819079157
117442 12781 825645212
109728 72676 494066640
13288 46196 356390165
119184 54172 673357109
20319 78740 8446459
31236 91334 601981085
94501 91464 712256853
67165 69365 178838866
99199 61143 816172210
86215 117888 747649348
120622 12636 660727131
18819 78760 297776406
26518 147809 188305184
50521 133571 927720297
130774 62287 264197923
20043 43168 822354804
99416 146164 563267875
147231 132798 140416145
52288 59816 549113803
35247 49672 873381977
102380 29346 551043472
83893 64431 992030683
25100 103466 773411163
29483 676 330464566
61235 34940 976811601
18341 18049 288082763
32742 114399 703049567
97724 116111 680267309
88882 17810 147951603
107758 66169 264314189
12438 143390 952012460
64309 120327 674411790
126366 51786 27820141
81503 114497 10401719
56623 85939 759791388
7986 15844 758211983
42477 132807 625351905
77733 113265 932202110
98838 107223 60645570
15888 69552 967699510
111954 57719 840709893
45298 38547 165538090
123246 145319 38820178
13441 138899 880696074
142704 78357 284995708
127778 139291 819573222
140888 10752 38369582
79744 134319 113421983
11629 6941 862797562
34501 147813 333529875
135607 137531 708751258
64416 147016 83494056
56605 18498 328598932
109838 81990 170252750
72923 144732 248675727
72366 83944 814784604
53432 78899 203140887
116946 139412 723182330
125519 106635 539092181
93656 94418 967296963
79011 34832 166300002
89367 75626 81389425
76966 146993 552899780
9945 135501 174188454
99686 60414 696745316
85092 75870 564622134
38961 2261 805517950
73948 119373 104025184
124420 12540 18112984
23245 31723 622349776
147846 31364 963865653
87260 89337 769786684
63420 103160 609006835
26857 52090 669075043
22687 15280 340955089
35069 37812 306331719
111375 110416 178897753
146596 19782 356378425
58041 44403 890813589
98438 73939 326699256
63610 119740 5096339
77518 79382 478340729
65680 73200 104059780
71346 91844 143103849
122344 98914 411681351
15448 104559 396226526
69716 9258 282081883
85118 70838 180892679
42189 17079 296444356
70681 74400 283069809
51146 81339 52522010
111589 99209 426983405
26602 97589 701760177
27580 57692 468146027
56420 135403 756344261
125964 126792 923265812
42310 86751 761291595
122823 47748 29617444
67815 48473 76723676
101379 15085 990845555
16381 44482 588174197
95740 118288 864163599
135910 141449 271934608
21107 125295 653599689
111533 34639 39982990
77719 148134 608968217
128800 128988 236129337
124226 128980 592559741
78379 91025 76276077
105237 98888 94360058
89187 84870 12876724
37771 134603 431310245
2545 118481 931157485
119677 116712 982649467
139129 15637 26945191
136582 91413 291210252
67539 36881 776814648
37488 69133 585578347
74553 5610 553052984
88195 107077 589398725
123477 129069 357880235
55353 20048 853838624
7846 114155 430069688
92305 95227 152583548
100420 45426 677472270
51468 83909 717416817
81277 108347 927053318
74203 94244 28788257
1294 86929 979722682
41586 79094 115251828
70449 39574 614541353
48624 106812 728851610
67650 78365 675724202
7071 133426 77411363
92679 17631 973841318
59083 112872 26392370
47616 10544 892629183
83510 18582 719585926
106203 123165 988382036
66815 5884 108187142
2562 125269 767682564
45393 32763 908887254
48100 39344 505560754
114596 116830 940358182
139848 94075 705944483
108359 43532 668092035
69692 113776 366210521
117614 13639 110304804
34690 50282 115420063
22851 126233 194731578
53791 107957 638025894
108578 54150 191472167
65202 113835 380473750
83539 31086 183483201
117112 4033 483110167
79664 73277 35244189
135919 120009 668856972
90092 138534 39523083
61525 12634 807924261
6724 59224 579120269
20847 77817 86199428
37815 101990 466007087
42329 141321 544115772
75659 129424 48696182
70196 23660 198835837
108494 98861 604454785
98892 59344 768136887
11374 18725 688712497
147074 119371 545515975
111268 96739 407532128
129943 19135 641121457
53011 114020 45348754
107980 90104 358426875
130443 111130 133666065
8943 139600 128295511
106627 23322 514731600
45794 128484 787658416
17141 90380 918263953
5521 69874 304613665
5855 112690 694439580
116431 19104 24430529
103693 125140 723391396
89974 113283 311952306
52944 30564 125596433
53722 13468 909392915
57610 114253 966149571
74942 67127 278471214
72453 126472 619235550
78721 135085 796440903
76231 66723 323940669
77955 82429 319381190
1365 135327 497344243
91123 79427 67161542
103057 35089 482761212
135178 27712 491423528
23796 33521 301598644
114489 22139 235988356
87680 56170 175022880
4412 89987 933043912
20382 113880 43643172
109373 103503 411042535
113341 70832 70885286
2985 16083 250475439
111507 75674 753818094
68840 106724 659381192
123935 145969 430910879
55543 46417 347259334
125440 15690 661376786
1415 144635 137244399
97838 74156 597784150
4703 108903 619766739
142543 49947 488006163
96870 2197 891690316
8980 68883 548663377
51041 99791 333250605
11219 52371 754924120
12458 94170 779600285
23626 29969 302722785
102430 6407 909431137
127032 146456 194367796
69785 51538 596902801
57015 92358 809212949
112749 95874 614653217
5780 34326 516821657
66824 46150 215834512
