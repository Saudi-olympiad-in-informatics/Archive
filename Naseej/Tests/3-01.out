19839765966325